scripts/deploy-resources.sh

```diff
@@ -31,6 +34,9 @@ oc apply -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_V
 
 oc patch kubevirt kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"developerConfiguration":{"featureGates": ["DataVolumes"]}}}}'
 
+# Deploy SSP
+oc apply -f "https://github.com/kubevirt/ssp-operator/releases/download/${SSP_OPERATOR_VERSION}/ssp-operator.yaml"
```

Comment:
can you please add a line which block the script until ssp is fully deployed?
e.g. something like:
`oc wait -n kubevirt deployment ssp-operator --for condition=Available --timeout 10m`
Do you need something special from ssp operator? If yes, you have to create ssp CR so ssp operator deploys its resources.